xbout/boutdataset.py

```diff
@@ -78,6 +79,81 @@ def getFieldAligned(self, name, caching=True):
                 self.data[aligned_name] = self.data[name].bout.toFieldAligned()
             return self.data[aligned_name]
 
+    def resetParallelInterpFactor(self, n):
+        """
+        Set the default factor to increase resolution when doing parallel interpolation.
+
+        Parameters
+        -----------
+        n : int
+            Factor to increase parallel resolution by
+        """
+
+        ds = self.data
+        ds.metadata['fine_interpolation_factor'] = n
+        for da in ds.values():
+            da.metadata['fine_interpolation_factor'] = n
+
+        return ds
+
+    def getHighParallelResVars(self, variables, **kwargs):
```

Comment:
I'll rename this as `interpolate_parallel`. I am inclined to leave `variables` as a required argument though, because I don't want to encourage using something like
```
ds_highres = ds.interpolate_parallel()
```
to get *all* of the variables in high resolution. That would also need to load all of the high resolution variables into memory! I think it's better to guide the user to pass only the specific subset of variables that they really want in high resolution.

Reply:
Fair enough. There should be a `variables='all'` special case though so I don't have to type out the whole list.

Reply:
Can we use something that's not a `str` for the special argument? It's unlikely that someone has a variable called `'all'` but not impossible. Would say `variables=True` be an abuse of `True`? It's not perfect, but I like that better than `variables=None` or `variables=False` for 'include everything'.

Reply:
How about using the `Ellipsis` built-in? Then the syntax for all variables would be `variables=...`. See for example in [this xarray PR].

Reply:
Yes, that reads right, I like it! :+1: